ampersand-form-view.js

```diff
@@ -183,7 +187,7 @@ module.exports = View.extend({
             delete this._startingValues;
         }
         this.handleSubmit = this.handleSubmit.bind(this);
-        this.el.addEventListener('submit', this.handleSubmit, false);
+        this.formEl.addEventListener('submit', this.handleSubmit, false);
```

Comment:
The only reason this is necessary is in cases where the root `el` is not the `<form>` tag (ie. it's inside a `<div>`) because the `submit` event does not get bubbled up beyond the `<form>`.

When you use your form view in your app, you shouldn't ever need to access `.formEl` because ampersand-form-view triggers the `submit` event on the view itself, so you'd listen to `formView.on('submit')`. If you need to do something with the DOM elements in the form view (say, append them to a container) you can still use `formView.el` as you normally would. `.formEl` is just for the HTML `submit` event.